### PR TITLE
PlayerInfo and map readability

### PR DESCRIPTION
**Pull request by rzulian**
Added buildings and gaiaformers
Change color for planets with building, mine as triangle, so triangle -> square+triangle -> square

### CODE REVIEW

File: src/components/Commands.vue [hunk 1]
```diff
@@ -253,7 +253,7 @@ export default class Commands extends Vue {
 
         case Command.Decline: {
           ret.push({
-            label: command.data.cost === '~' ? `Decline ${command.data.offer}` : `Decline ${command.data.offer} for ${command.data.cost}`,
+            label: `Decline ${command.data.offer}`,
             command: Command.Decline
```
Comment (coyotte508) [inline src/components/Commands.vue]: this can be command: `${Command.Decline} ${command.data.offer}`, so logs are more readble

**Comment by coyotte508** [comment]
I'll check the visuals this evening, it's hard to picture without a screenshot. I don't know about the circle, for example what happens with gaia-formers & transdim planets.

File: src/components/Commands.vue (hunk 1 above)
Comment (rzulian) [reply src/components/Commands.vue]: Isn't the same result?

**Comment by coyotte508** [comment]
image

The mine is a rectangle, everything fits the current visuals except academies, which we could change to be a circle with a rectangle part. The trading station should be moved down on the Y axis a bit too probably.

I'm not sure why we can't leave the planets be and need to overwrite them. This isn't like TM where you put a tile over the terrain, in GP you just put buildings.

We could add the info about the player occupying the planet in the tooltip (where we show coordinates & sector & building).

**Comment by coyotte508** [comment]
I really do prefer the first, IMO it gives a better overview.

The player can easily be given a choice with a single css class, as long as we keep the same buildings. We can have a toggle / an option setting in a floating sidebar, which will just activate fill in the buildings & disable the circles. 

As for which option to activate by default, we can let the feedback from the players decide.

There' still two problems though:

- Ivit's space stations
- Transdim gaia-formers

With your current PR, the players don't know if the planet they're on has been gaia-formed yet or not.

**Comment by rzulian** [comment]
We can test different solutions, and ask players what they prefer. First feedback we got from the play test was that the original was difficult, especially because mine icon is small and the real faction color is hardly visible, especially with legacy display. I didn't try with iPad to check on small screens though.

Regarding the original planet, my idea is to use the steps to represent what you have. I've already added the ownedPlanets to the json. I've tried to modify steps but I wasn't able.

File: src/components/Commands.vue (hunk 1 above)
Comment (coyotte508) [reply src/components/Commands.vue]: the log would show `decline 2pw` or `decline up-nav`, instead of just `decline`

**Pull request merged**